nbs/10_compute_correlations/05_gtex_v8/py/07_01-gtex-var_pc_log2-clustermatch.py

```diff
@@ -27,19 +27,30 @@
 
 # %% tags=[]
 import pandas as pd
+from tqdm import tqdm
 
 from clustermatch import conf
-from clustermatch.corr import spearman
+from clustermatch.utils import simplify_string
+from clustermatch.corr import clustermatch
 
 # %% [markdown] tags=[]
 # # Settings
 
 # %% tags=[]
-# we don't have gene subsets for recount2
-# GENE_SELECTION_STRATEGY = "var_raw"
+GENE_SELECTION_STRATEGY = "var_pc_log2"
 
 # %% tags=[]
-CORRELATION_METHOD = spearman
+# for clustermatch, I select the top 5 tissues (according to sample size, see nbs/05_preprocessing/00-gtex_v8-split_by_tissue.ipynb)
+TISSUES = [
```

Comment:
Maybe you could save these somewhere rather than hardcoding them? I guess it's unlikely that you'll decide to use a different version of GTEx, but if you did you'd have to track this down and edit it by hand.

Reply:
I thought exactly the same at some point :laughing: However, we are specifically working with GTEx v8 (the latest version of GTEx), so this is very unlikely to change, we won't use a previous version. I'd prefer to keep the tissues here to avoid the extra work.